blueflood-elasticsearch/src/main/resources/events_mapping.json

```diff
@@ -0,0 +1,27 @@
+{
+  "graphite_event": {
+    "_routing": {
+      "required": true
+    },
+    "properties": {
+      "tenantId": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "what": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "data": {
+        "type": "string",
+        "index": "not_analyzed"
+      },
+      "tags": {
+        "type": "string"
```

Comment:
@tilogaat Yes, but for the tags we have not specified `not_analyzed` and I think the default is that ES will analyze these fields. And by analyze I mean it will tokenize the field and use the default filter while searching the data. Do we want this extra functionality for `tags`?